src/wtf/backend.cc

```diff
@@ -145,6 +145,25 @@ bool Backend_t::SimulateReturnFromFunction(const uint64_t Return) {
   return true;
 }
 
+bool Backend_t::SimulateReturnFrom32bitFunction(const uint32_t Return, const uint32_t stdcallArgsCount) {
```

Comment:
nit: do you mind changing `stdcallArgsCount` into `StdcallArgsCount`?

Reply:
done it